Package.swift

```diff
@@ -1,16 +1,17 @@
-// swift-tools-version: 5.10
```

Comment:
How much work on support Swift 5.10 again?

Reply:
Parameter pack iteration is not available in 5.10. Using compiler very checks to get around